locales/pt-BR/client.json

```diff
@@ -1398,7 +1398,7 @@
   "SUBSCRIPTION_LIST.CancelExtrasDate": "Cancele a sua subscrição Extra quando esta terminar em {{date}}",
   "SUBSCRIPTION_LIST.CancelForgeBeforeStartDate": "Cancele a sua assinatura da Forge antes de esta começar em {{date}}",
   "SUBSCRIPTION_LIST.CancelForgeWhenEndDate": "Cancele a sua assinatura da Forge quando esta terminar em {{date}}",
-  "SUBSCRIPTION_LIST.PlanExpiresDate": "Expira em {{date}}",
+  "SUBSCRIPTION_LIST.PlanExpiresDate": "Expira em {{expirationDate}}",
```

Comment:
This is wrong! The variable is actually `date`:
`{{t "SUBSCRIPTION_LIST.PlanExpiresDate" date=expirationDate}}`